Charts: update RangeBar demo description (#2214)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/RangeBar/description.md

This demo illustrates the range bar [series type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). 

> You can use range bars to display value ranges that correspond to specified arguments. Each point in a range bar series has one argument and two values. The [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#argumentField) supplies arguments; the [rangeValue1Field](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#rangeValue1Field) and [rangeValue2Field](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#rangeValue2Field) supply values - starting and ending points of bars.

Reviewer: The simpler your grammar and punctuation are, the better.

```suggestion
You can use range bars to display value ranges that correspond to specified arguments. Each point in a range bar series has one argument and two values. The [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#argumentField) supplies arguments while the [rangeValue1Field](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#rangeValue1Field) and [rangeValue2Field](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#rangeValue2Field) supply bar start and end point values.
```

> To create multiple range bar series, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array to declare each series and the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify the common series type. In that object you can implement specific parameters to configure [range bar](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/) specifically. For example, this demo specifies a [minimum bar size](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#minBarSize) for all series.

Reviewer: > In that object you can implement specific parameters to configure [range bar](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/) **specifically**.

1. An article is missing (range bar).
2. What do you mean by "specifically" here exactly? To configure a specific range bar? Or something else, perhaps?

Author: So, the **commonSeriesSettings** object has common Chart parameters and specific parameters for each series type. I just wanted to emphasize that this settings are specifically designed for this type. Maybe the following sentence is better?

> In that object you can implement parameters specific to [range bar series](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/) type.

Reviewer: Indeed, this version makes more sense, however, you've lost an article again... Which is regrettable.

> In this object

A clause separated by a comma. It's better to say "this" because we do not need to differentiate two similar objects "this/that". However, you do realize that it's better to avoid such scenarios in documentation when we have two differentiate things using similar pronouns.

```suggestion
To create multiple range bar series, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array to declare each series and the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify the common series type. In this object, you can implement parameters specific to the [range bar series](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/) type. For example, this demo specifies a [minimum bar size](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/RangeBarSeries/#minBarSize) for all series.
```
